### DISCUSSION TITLE
Multiple PN532 Readers

### DISCUSSION THREAD

**Discussion by Maxime0506** [rednblkx/HomeKey-ESP32#49] [Q&A] [answered]
Hello I wanted to ask if it is possible to connect to separate PN532 Readers to the same ESP32 for 2 separate looks and also that they can booth be used without the other one opening

**Comment by ihrapsa** [comment, marked as answer]
What usecase do you have in mind? 🤔
As far as I know the **esp32-pn532** combo is independent of the lock and the unlock logic is done using an automation service like Home Assistant or NodeRed based on the mqtt messages that the ESP32 broadcasts.

In other words, a single reader can be used to control as many locks as you want.

Using two or more readers with the esp32 is definitely possible over SPI, although that’s not implemented yet in this project’s code.
